### PR TITLE
[cilium] Add Dockerfile

### PR DESCRIPTION
**Pull request by AdamKorcz**
This Dockerfile finalizes initial integration of Cilium.

Previous steps of the integration were taken [here](https://github.com/cilium/cilium/pull/14202) and [here](https://github.com/google/oss-fuzz/pull/4784).

Fuzzer is currently still pending upstream which explains the failing build.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Can you add the build.sh as well?
I don't think we will merge this without the build.sh.

**Comment by AdamKorcz** [comment]
The build file is added upstream and is handled in the same way as https://github.com/google/oss-fuzz/tree/master/projects/go-ethereum. Including it in the project folder is not necessary.

**Comment by jonathanmetzman** [comment]
> The build file is added upstream and is handled in the same way as https://github.com/google/oss-fuzz/tree/master/projects/go-ethereum. Including it in the project folder is not necessary.

Ah my bad.

**Comment by AdamKorcz** [comment]
The failed coverage build does not reproduce locally.

**Comment by AdamKorcz** [comment]
@inferno-chromium @jonathanmetzman @oliverchang Ready for review

**Comment by AdamKorcz** [comment]
@jonathanmetzman I fixed the wget error.

File: projects/cilium/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC.
```
Comment (jonathanmetzman) [inline projects/cilium/Dockerfile:1]: Nit: no period at the end of the copyright line.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**